dashboard/test/models/user_test.rb

```diff
@@ -1090,7 +1096,7 @@ def assert_fails_email_uniqueness_validation(user)
 
     assert(user_level1.updated_at < user_level2.updated_at)
 
-    next_script_level = user.next_unpassed_progression_level(script)
+    next_script_level = user.next_unpassed_visible_progression_level(script)
     refute next_script_level.nil?
```

Comment:
This test doesn't pin down that the desired behavior is.  Is `next_script_level` supposed to be the other multi sublevel, or the new maze level you've added?  If it's the maze level, how was this test passing before?

Reply:
I added the maze level because without it, the script was complete so the method was returning nil.

Reply:
I think that means the definition of a script being complete has changed with regard to level groups.  Let's make sure that we understand what changed about that behavior and document it.